packages/sanity/src/core/search/weighted/createSearchQuery.test.ts

```diff
@@ -27,15 +32,14 @@ describe('createSearchQuery', () => {
         `// findability-mvi:${FINDABILITY_MVI}\n` +
```

Comment:
I guess we can/should remove these now? cc @robinpyon - have these been used for anything other than debugging?